PushNotifications.md

```diff
@@ -210,6 +210,12 @@ data: 'This is a Ably message published on channels that is also '
 }));
 ```
 
+#### Prioritising messages
+
+Only use high priority when it requires immediate user attention/ interaction. Use the normal priority (5) otherwise. Messages with priority wake a device from a battery saving state, using more of the user device battery.
```

Comment:
```suggestion
Only use high priority when it requires immediate user attention or interaction. Use the normal priority (5) otherwise. Messages with a high priority wake a device from a battery saving state, which drains the battery even more.
```